desktop-widgets/locationinformation.cpp

```diff
@@ -219,6 +221,21 @@ void LocationInformationWidget::initFields(dive_site *ds)
 	}
 }
 
+
+void LocationInformationWidget::on_GPSbutton_clicked()
+{
+	QString fileName = QFileDialog::getOpenFileName(  this, tr("Select GPS file to open"),
```

Comment:
`getOpenFileName(  this...` -> `getOpenFileName(this...`

Reply:
done.